Muscle_Driver/muscle.py

```diff
@@ -78,7 +92,22 @@ def triggered(self):
                 self.grip_T0 = time.time()
                 return False
     
-    #hey Jered, this code is meant to be run in a loop. Am I writing this correctly?
+
+    def bufferedTrigger(self):
+        #create buffers, take mean, see if next buffer is greater by a certain value
+        for i in range(len(self.currentBufferList)):
+            self.currentBufferList[i] = self.AnalogRead()
```

Comment:
Does this need to have a sleep() to ensure that the analogRead has changed values recently?

Reply:
no it reads the value on the adc in real time as fast as it can. the less delay the better

Reply:
OHH I just understood your comment. Nah it seems to be working fine when I tested, here are some results

![image](https://user-images.githubusercontent.com/40764347/116507948-24626400-a875-11eb-8cee-aee2d7f07f7a.png)